depc/controllers/dependencies.py

```diff
@@ -302,3 +424,71 @@ def _build_dependencies_query(
         return query.format(
             where=where, order=order, topic=topic, label=label, name=node
         )
+
+    @classmethod
+    def _build_impacted_nodes_queries(
+        cls, topic, label, node, impacted_label, skip=None, limit=None, count=False
+    ):
+        # Get the standard impacted nodes query
+        query = "MATCH p = (n:{topic}_{impacted_label})-[*]->(:{topic}_{label}{{name: '{name}'}}) WITH *, relationships(p) AS r_list WITH *, nodes(p) as n_sub_list RETURN DISTINCT n AS impacted_node, collect({{ relationships: r_list, nodes: n_sub_list }}) AS all_path_elements ORDER BY n.name SKIP {skip} LIMIT {limit}".format(
```

Comment:
Is-it possible to display the query like here please (https://github.com/ovh/depc/blob/2eae502b442bdf81fe849e7fe880471087013973/depc/controllers/dependencies.py#L40) ?